docs/podman.1.md

```diff
@@ -35,6 +35,11 @@ Path to where the cpu performance results should be written
 
 Log messages above specified level: debug, info, warn, error (default), fatal or panic
 
+**--max-workers**
+
+When certain operations can be done in parallel, limit the number of parallel operations to a given integer. Examples
+where parallel operations are used are podman rm, podman stop, and podman ps.
```

Comment:
It would be good to specify what the default value is if not specified.

Reply:
fixed